com.github.zyedidia.micro.yml

```diff
@@ -0,0 +1,93 @@
+app-id: com.github.zyedidia.micro
```

Comment:
app-id must start with `io.github` as per the docs.

Reply:
Now it is `io.github.zyedidia.micro`